students/shibin_mathew/lesson3/list_lab.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+#Series 1
+list_ = ["Apples","Pears","Oranges","Peaches"]
+print(list_)
+print("Please add one more fruit to the list")
+response = input()
+list_.append(response)
+print(list_)
+print("Please enter a number")
+response_num = input()
+print(response_num," ",list_[int(response_num)+1])
+list_=["Bananas"]+list_
+print(list_)
+list_.insert(0,"Grapes")
+print(list_)
+for i in list_:
+	if i[0] == 'P':
+		print(i)
+
+
+#Series 2
+print(list_)
+list_=list_[:-1]
+print(list_)
+print("What fruit would you like to remove")
+response = input()
+if response in list_:
+	list_.remove(response)
+print(list_)
+
+#Series 3
+bool = False
```

Comment:
be careful about overwriting built-ins, here `bool` is being overwritten (notice github special color highlight)